config/cscs.py

```diff
@@ -973,7 +973,7 @@
                 'eiger', 'pilatus'
             ],
             'modules': [
-                {'name': 'PrgEnv-aocc', 'collection': True}
+                {'name': 'PrgEnv-aocc', 'collection': False}
```

Comment:
There is no need for the extended syntax. Simply write it as `'modules': ['PrgEnv-aocc']` etc.